web-ui/src/pages/CheckinsReportPage.jsx

```diff
@@ -1,149 +1,161 @@
-import React, { useContext, useEffect, useRef, useState } from 'react';
-
-import { TextField } from '@mui/material';
-import Autocomplete from '@mui/material/Autocomplete';
+import React, { useContext, useEffect, useState } from 'react';
 
 import { AppContext } from '../context/AppContext';
-import CheckinReport from '../components/reports-section/CheckinReport';
 import {
   selectCheckinPDLS,
-  selectTeamMembersWithCheckinPDL
+  selectTeamMembersWithCheckinPDL,
+  selectMappedPdls
 } from '../context/selectors';
-import { isArrayPresent } from '../helpers/checks';
-import { useQueryParameters } from '../helpers/query-parameters';
+
+import {
+  Grid,
+  Typography,
+  IconButton,
+  Box,
+  ButtonGroup,
+  Tooltip
+} from '@mui/material';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+
+import CheckinReport from '../components/reports-section/CheckinReport';
+import MemberSelector from '../components/member_selector/MemberSelector';
+import { FilterType } from '../components/member_selector/member_selector_dialog/MemberSelectorDialog';
+import { getQuarterDuration } from '../helpers/datetime';
 
 import './CheckinsReportPage.css';
+/**
+ * Sort Members by the last name extracted from the full name.
+ * @param {MemberProfile} a - First Member object
+ * @param {MemberProfile} b - Second Member object
+ * @returns {number} - Comparison result for sorting
+ */
+function sortByLastName(a, b) {
+  if (!a.name || !b.name) return;
```

Comment:
I think the `sort` method expects this function to return a number. Maybe return zero here.

Reply:
TIL. Thanks, Mark! Done.